src/content/docs/en/guides/integrations-guide/svelte.mdx

```diff
@@ -103,44 +103,28 @@ To use your first Svelte component in Astro, head to our [UI framework documenta
 
 This integration is powered by `@sveltejs/vite-plugin-svelte`. To customize the Svelte compiler, options can be provided to the integration. See the [`@sveltejs/vite-plugin-svelte` docs](https://github.com/sveltejs/vite-plugin-svelte/blob/HEAD/docs/config.md) for more details.
 
-### Default options
```

Comment:
Note there are now no default options set by Astro. It falls back to the behaviour in `@sveltejs/vite-plugin-svelte` by default now.

Which means the notes about `vitePreprocess` are no longer relevant, the user will have to always add that themselves if they use scss/stylus etc. Since `astro add svelte` has been adding those for users by default for a long time, I don't think this will break a lot of folks. I've noted this in the core PR changeset